eng/pipelines/templates/Apex_Tests_On_Windows.yml

```diff
@@ -41,11 +41,12 @@ stages:
           - ${{parameters.variables}}
       runSettingsURI: ${{parameters.runSettingsURI}}
       visualStudioBootstrapperURI: $(bootstrapperUrl)      
-      baseVisualStudioBootstrapperURI: ${{parameters.baseBuildDrop}};bootstrappers/Enterprise/vs_enterprise.exe
```

Comment:
Can we `baseBuildDrop` parameter from the template?

Reply:
`baseVisualStudioBootstrapperURI: https://vsdrop.corp.microsoft.com/file/v1/$(VisualStudio.BaseBuild.ProductsDropName);bootstrappers/Enterprise/vs_enterprise.exe`

In `baseVisualStudioBootstrapperURI`, everything is constant except `$(VisualStudio.BaseBuild.ProductsDropName)` which is retrieved from `SetVisualStudioBaseBuildProductsDropName` or `SetBaseProductsDropNameToTarget`. So, it can't be passed using `baseBuildDrop`

Reply:
Thanks for the clarification. I am referring to `baseBuildDrop` parameter itself. I think we can delete the said parameter as it is no longer needed. In other words, there are several references to this parameter in the repo, https://github.com/search?q=repo%3ANuGet%2FNuGet.Client%20baseBuildDrop&type=code in the repository. Can we remove this parameter from the code?